esp-hal/src/spi/master.rs

```diff
@@ -1579,6 +1579,12 @@ mod dma {
                 tx_buf,
             }
         }
+
+        /// Splits `SpiDmaBus` back into `SpiDma` to allow for manual
+        /// buffer control
```

Comment:
Square braces are understood properly as references.
```suggestion
        /// Splits [SpiDmaBus] back into [SpiDma], [DmaRxBuf] and [DmaTxBuf].
```

Reply:
Fixed. Btw the `new()` also has backticks rather than brackets (which is why I used it like that)

Reply:
Yeah I spotted that. We can attend to that separately. Couldn't find anything about this in the API-GUIDELINES